Add note about increased window and interval can cause WiFi instability (#2790)
OttoWinter/esphomedocs: components/bluetooth_proxy.rst

      scan_parameters:
        interval: 1100ms
        window: 1100ms
.. note::

>     For WiFi-based proxies, changing the interval or window from their default values may result in an unstable WiFi connection. Keep the default values for interval and window will usually resolve any instability.

Author: ```suggestion
    For WiFi-based proxies, changing the interval or window from their default values may result in an unstable WiFi connection. Keeping the default values for interval and window will usually resolve any instability.
```

>     For WiFi-based proxies, changing the interval or window from their default values may result in an unstable WiFi connection. Keeping the default values for interval and window will usually resolve any instability.

Author: ```suggestion
    For WiFi-based proxies, changing the `interval` or `window` from their default values may result in an unstable WiFi connection. Using the default values for `interval` and `window` will usually resolve any instability.
```

>     For WiFi-based proxies, changing the `interval` or `window` from their default values may result in an unstable WiFi connection. Using the default values for `interval` and `window` will usually resolve any instability.

Author: ```suggestion
    For WiFi-based proxies, changing the ``interval`` or ``window`` from their default values may result in an unstable WiFi connection. Using the default values for ``interval`` and ``window`` will usually resolve any instability.
```

      scan_parameters:
        interval: 1100ms
        window: 1100ms

>         
> For WiFi-based proxies, 1100ms can result in an unstable WiFi connection. Setting the default values for interval and window will usually resolve any instability.

Reviewer: ```suggestion
.. note::

    For WiFi-based proxies, changing the interval or window from their default values may result in an unstable WiFi connection. Keep the default values for interval and window will usually resolve any instability.
```

This configuration is for an Olimex ESP32-PoE-ISO board with an Ethernet connection to the network. If you use a different board, you must change the ``board`` substitution to match your board.

For WiFi-based proxies, 1100ms can result in an unstable WiFi connection. Setting the default values for interval and window will usually resolve any instability.

> 

Reviewer: Not sure if we should repeat this?

Instead, maybe change the header to:
```
Complete sample recommended configuration for an ethernet connected Bluetooth proxy
-----------------------------------------------------------------------------------
```

Author: adjusted 👍
